src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@mixin cell-propertis ($cell-width, $cell-height, $bg-color) {
+  width: $cell-width;
+  height: $cell-height;
+  background-color: $bg-color;
+}
+
+$space-size: 1px;
+$color-hover: pink;
+$start-day:
+  mon 1,
+  tue 2,
+  wed 3,
+  thu 4,
+  fri 5,
+  sat 6,
+  sun 7;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space-size;
+  width: 720px;
+  margin: 0 auto;
+  padding: 10px;
+
+  &__day {
+    @include cell-propertis(100px, 100px, #eee);
+    border: $space-size solid black;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 500ms;
+  }
+
+  &__day:hover {
+    background-color: $color-hover;
+    cursor: pointer;
```

Comment:
Why alter the cursor property on hover?

Reply:
I made it according to the instruction.
**On hovering over a cell:

cursor should become pointer
The hovered cell has to become pink**